crates/hc_sandbox/src/calls.rs

```diff
@@ -418,64 +417,82 @@ async fn call_inner(client: &mut AdminWebsocket, call: AdminRequestCli) -> anyho
                     },
                 }])
                 .await?;
-            msg!("Added admin port {}", port);
+            println!("{}", serde_json::to_value(port)?);
         }
         AdminRequestCli::AddAppWs(args) => {
             let port = args.port.unwrap_or(0);
             let port = client
                 .attach_app_interface(port, args.allowed_origins, args.installed_app_id)
                 .await?;
-            msg!("Added app port {}", port);
+            println!("{}", serde_json::to_value(port)?);
         }
         AdminRequestCli::ListAppWs => {
             let interface_infos = client.list_app_interfaces().await?;
-            msg!("Attached app interfaces {:?}", interface_infos);
+            println!("{}", serde_json::to_value(&interface_infos)?);
         }
         AdminRequestCli::RegisterDna(args) => {
-            let dnas = register_dna(client, args).await?;
-            msg!("Registered DNA: {:?}", dnas);
+            let dna = register_dna(client, args).await?;
+            println!("{}", serde_json::to_value(dna.to_string())?);
         }
         AdminRequestCli::InstallApp(args) => {
             let app = install_app_bundle(client, args).await?;
-            msg!("Installed app: {}", app.installed_app_id,);
+            println!("{}", app_info_to_json(app)?);
         }
         AdminRequestCli::UninstallApp(args) => {
             client
                 .uninstall_app(args.app_id.clone(), args.force)
                 .await?;
-            msg!("Uninstalled app: {}", args.app_id,);
+            msg!("Uninstalled app: \"{}\"", args.app_id);
         }
         AdminRequestCli::ListDnas => {
-            let dnas = client.list_dnas().await?;
-            msg!("DNAs: {:?}", dnas);
+            let dnas: Vec<DnaHashB64> = client
+                .list_dnas()
+                .await?
+                .into_iter()
+                .map(|d| d.into())
+                .collect();
+            println!("{}", serde_json::to_value(&dnas)?);
         }
         AdminRequestCli::NewAgent => {
             let agent = client.generate_agent_pub_key().await?;
-            msg!("Added agent {}", agent);
+            println!("{}", serde_json::to_value(agent.to_string())?);
         }
         AdminRequestCli::ListCells => {
-            let cells = client.list_cell_ids().await?;
-            msg!("Cell IDs: {:?}", cells);
+            let cell_id_values: Vec<serde_json::Value> = client
+                .list_cell_ids()
+                .await?
+                .into_iter()
+                .map(serde_json::to_value)
+                .collect::<Result<Vec<serde_json::Value>, serde_json::Error>>()?;
+            let converted = cell_id_values
+                .into_iter()
+                .map(cell_id_json_to_base64_json)
+                .collect::<anyhow::Result<Vec<serde_json::Value>>>()?;
```

Comment:
This is library code. We used to use `anyhow` pretty heavily in library code but it is needlessly hard to integrate with other error handling libraries when this code is used by other applications. Prefer using an existing error type where possible, or use `thiserror` to create a new error type otherwise.